api/net/Areas/Editor/Controllers/LookupController.cs

```diff
@@ -124,6 +133,16 @@ public async Task<IActionResult> FindAllAsync()
         if (String.IsNullOrWhiteSpace(_CssOptions.ClientId)) throw new ConfigurationException("CSS clientId has not been configured");
         if (String.IsNullOrWhiteSpace(_CssOptions.Secret)) throw new ConfigurationException("CSS secret has not been configured");
 
+        CanadaHolidayModel? statHolidays = null;
+        try
+        {
+            statHolidays = await _httpClient.GetAsync<CanadaHolidayModel>("https://canada-holidays.ca/api/v1/provinces/BC");
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Failed to fetch BC holidays");
```

Comment:
We don't want a failure on the government site to break our app.  It will simply result in an empty array of holidays.